numpy/core/src/multiarray/compiled_base.c

```diff
@@ -1540,62 +1524,34 @@ pack_inner(const char *inptr,
     npy_intp index = 0;
     int remain = n_in % 8;              /* uneven bits */
 
-#if defined NPY_HAVE_SSE2_INTRINSICS && defined HAVE__M_FROM_INT64
+#if NPY_SIMD
```

Comment:
@mattip, this guard will activated only, if there are enaabled baseline features supported by `NPYV`

Reply:
right, we compile on a machine that has those features, so the binary we supply for general use will have a baseline profile.